package.json

```diff
@@ -37,7 +37,7 @@
     "gulp-stylint": "^4.0.1",
     "gulp-stylus": "^2.7.0",
     "gulp-svgstore": "^6.1.1",
-    "kss": "^3.0.0-beta.18",
+    "kss": "3.0.0-beta.18",
```

Comment:
Why?

Reply:
3.0.0-beta.19 is out and i break something in it
https://github.com/kss-node/kss-node/issues/469